### PR TITLE
[15.0][MIG] product_supplierinfo_for_customer_picking

### PR DESCRIPTION
**Pull request by JuanyDForgeflow**
Based on product_supplierinfo_for_customer, this module loads in every Stock move the
customer code defined in the product.

### CODE REVIEW

**Comment by rousseldenis** [comment]
@JuanyDForgeflow Could you fix tests ? Thanks

File: product_supplierinfo_for_customer_picking/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2013 - 2021 Agile Business Group sagl (<https://www.agilebg.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Product Supplierinfo For Customer Picking",
+    "version": "15.0.1.0.1",
```
Comment (MiquelRForgeFlow) [inline product_supplierinfo_for_customer_picking/__manifest__.py]:
```suggestion
    "version": "15.0.1.0.0",
```

**Comment by lfreeke** [comment]
When I have more than one customer in the supplier info customer list
image

The Customer Product Code from the first customer is set on the the stock move and not the Customer Product Code from the customer you have selected in the Sale Order. 

_Sale Order_:
image

_Stock Move_:

image

**Comment by rousseldenis** [comment]
@JuanyDForgeflow Could you please ping author of #1016 in order to supersedes ?

**Comment by JuanyDForgeflow** [comment]
@rousseldenis If the other pr is already working I think It's best we close this one, what do you think?

**Comment by rousseldenis** [comment]
> @rousseldenis If the other pr is already working I think It's best we close this one, what do you think?

@JuanyDForgeflow I think we can close the other as no answer and owner has no github activity

**Comment by rousseldenis** [comment]
@JuanyDForgeflow Could you look at #1016 on migration commit ? I think there are clues to fix tests. Thanks

**Review by JasminSForgeFlow — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
Functional review, with several customers on the same product

**Comment by AaronHForgeFlow** [comment]
@lfreeke the commented issue is now fixed. I wonder if you can help here with a new review.

**Review by LoisRForgeFlow — APPROVED** [review]
LG!

**Pull request merged**